### PR TITLE
Just a first sketch for start.

### PR DESCRIPTION
**Pull request by natlibesa**
Just for the start, first steps: field 008 when used in boardgames needs checking.

### CODE REVIEW

**Comment by natlibesa** [comment]
Now with board game check & changes made with fill.

File: src/transform.js [hunk 1]
```diff
@@ -191,6 +190,17 @@ export default function (stream, {validate = true, fix = true}) {
 					chars[39] = 'c';
 				}
 
+				// Check: is it a boardgame? - >  000/06 = 'r'  &   008/33 = 'g'
+				if ((marcRecord.leader[6] === 'r') & (chars[33] === 'g')) {
```
Comment (natlibfi-arlehiko) [inline src/transform.js:194]: Inner parentheses are superfluous. Remove.

File: src/transform.js [hunk 2]
```diff
@@ -191,6 +190,17 @@ export default function (stream, {validate = true, fix = true}) {
...
+					console.log(' NOW IT IS A BOARDGAME! '); // Boardgame, so do edits:
+					chars.fill('|', 18, 19);
+					chars.fill('|', 19, 20);
+					chars.fill('|', 20, 21);
+					chars.fill('|', 30, 31);
+					chars.fill('|', 31, 32);
+					chars.fill('|', 34, 35);
+				}
+
```
Comment (natlibfi-arlehiko) [inline src/transform.js:203]: - Remove debug logging
- Only one `.fill` call is necessary. Refer to the [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/fill).

Comment (natlibesa) [reply src/transform.js]: Checked things above (v. 6.5.2020)

**Pull request merged**